net/freeradius3/opnsense/mvc/app/models/OPNsense/Freeradius3/Menu/Menu.xml

```diff
@@ -1,5 +1,7 @@
 <menu>
-  <Routing cssClass="fa fa-map-signs" order="45">
-    <General VisibleName="General" cssClass="fa fa-cog fa-fw" url="/ui/freeradius3/general/index" order="1"/>
-  </Routing>
+  <Services>
+    <FreeRADIUS VisibleName="FreeRADIUS" cssClass="fa fa-cog fa-fw">
+      <General VisibleName="General" cssClass="fa fa-cog fa-fw" url="/ui/freeradius3/general/index" order="1"/>
```

Comment:
Do you plan to use the extra navigation depth soon? otherwise you can make FreeRADIUS the linked item